### PR TITLE
Limit the blog post title length to 3 rows ~70 characters

### PR DESCRIPTION
**Pull request by DongreJaipal**
## This PR Closes Issue closes #
https://github.com/AnguHashBlog/angular-primeng/issues/30

## Description
1.Limit the blog post title length to 3 rows ~70 characters
2.Created a custom pipe name (truncate) ,takes  input (optional) for truncating the string to the desired length, and at the at the end of the string, added "..." .
3.used the truncate pipe in series and post components for post title.

## What type of PR is this? (check all applicable)

- [✓] 🅰️ Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings
[Attach screenshots or recordings if applicable]

## Added to documentation?
- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

## [Optional] Post-deployment tasks
[Specify any post-deployment tasks that need to be performed]

## [Optional] What gif best describes this PR or how it makes you feel? 
[Embed gif or describe the feeling in plain text]

### CODE REVIEW

File: angular-primeng-app/src/app/components/series/series.component.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { ButtonModule } from "primeng/button";
 @Component({
   selector: 'app-series',
   standalone: true,
-  imports: [RouterLink, AsyncPipe, CardModule, InfiniteScrollDirective, ButtonModule],
+  imports: [RouterLink, AsyncPipe, CardModule, InfiniteScrollDirective, ButtonModule,CommonModule],
   templateUrl: './series.component.html',
```
Comment (cleancodecraft) [inline angular-primeng-app/src/app/components/series/series.component.ts:14]: @DongreJaipal ok things look good up to here

File: angular-primeng-app/src/app/pipes/truncate.pipe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { TruncatePipe } from './truncate.pipe';
+
```
Comment (cleancodecraft) [inline angular-primeng-app/src/app/pipes/truncate.pipe.spec.ts:2]: @DongreJaipal please remove this file the - `angular-primeng-app/src/app/pipes/truncate.pipe.spec.ts`

File: angular-primeng-app/src/app/pipes/truncate.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
```
Comment (cleancodecraft) [inline angular-primeng-app/src/app/pipes/truncate.pipe.ts:2]: @DongreJaipal please remove this file the - `angular-primeng-app/src/app/pipes/truncate.pipe.ts` 

the two files for the pipe should not be included the rest is ok

**Review by cleancodecraft — CHANGES_REQUESTED** [review]

File: angular-primeng-app/src/app/pipes/truncate.pipe.ts (hunk 3 above)
Comment (DongreJaipal) [reply angular-primeng-app/src/app/pipes/truncate.pipe.ts:2]: okay @cleancodecraft

**Review by monacodelisa — APPROVED** [review]

**Review by copypasteov — APPROVED** [review]

**Review by cleancodecraft — APPROVED** [review]

**Comment by monacodelisa** [comment]
@DongreJaipal thank you :)

**Pull request merged**